bridges/T411Bridge.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * T411Bridge
+ * Returns 5 newest torrents with specified search criteria
+ *
+ * @name T411
+ * @homepage https://t411.in/
+ * @description Returns the 5 newest torrents with specified search terms <br /> Use url part after '?' mark when using their search engine
+ * @maintainer ORelio
+ * @update 2015-10-22
+ * @use1(search="search criteria")
+ */
+class T411Bridge extends BridgeAbstract {
+
+    public function collectData(array $param) {
+
+        //Utility function for extracting CDATA fields
+        function StripCDATA($string) {
+            $string = str_replace('<![CDATA[', '', $string);
+            $string = str_replace(']]>', '', $string);
+            return $string;
+        }
+
+        //Utility function for removing text based on specified delimiters
+        function StripWithDelimiters($string, $start, $end) {
+            while (strpos($string, $start) !== false) {
+                $section_to_remove = substr($string, strpos($string, $start));
+                $section_to_remove = substr($section_to_remove, 0, strpos($section_to_remove, $end) + strlen($end));
+                $string = str_replace($section_to_remove, '', $string);
+            } return $string;
+        }
+
+        //Ensure proper parameters have been provided
+        if (empty($param['search'])) {
+            $this->returnError('You must specify a search criteria', 400);
+        }
+
+        //Retrieve torrent listing as truncated rss, which does not contain torrent description
+        $url = 'http://www.t411.in/torrents/rss/?'.$param['search'].'&order=added&type=desc';
```

Comment:
After some investigation, it seems like their RSS feed feature only works properly for recent content. Eg if I search for `ubuntu`, the most recent item is 5 months old, and rss feed for that search does not match, displaying first an item published in 2011. However, if I search for, let's say, some popular category like the `movie` category, I get the same exact results from the search results and the rss feed, with some content added about 1 hour ago. That should explain why I never saw the issue you encountered while developping this bridge, because I tested search queries giving out some recent content.